cylc/flow/scripts/scan.py

```diff
@@ -175,14 +181,24 @@ def get_option_parser():
     parser.add_option(
         '--states',
         help=(
-            'Choose which flows to display by providing a list of states'
+            'Choose which workflows to display by providing a list of states'
             ' or "all" to show everything. See the full `cylc scan` help'
             ' for a list of supported states.'
         ),
         default='running,paused,stopping',
         action='store'
     )
 
+    parser.add_option(
+        '--uninstalled', '-u',
+        help=(
+            'List uninstalled workflows from configured source directories.'
+            ' (overrides the --states option)'
+        ),
```

Comment:
I wonder if "uninstalled" is the right word? Technically they're non-installed workflows if they haven't been installed at all yet. Perhaps `--source`, `--src` would be more appropriate?

Reply:
Yeah, `--source, -s` makes sense.